src/storage/storage.ts

```diff
@@ -1,44 +1,39 @@
-import { Config, getConfig } from "../config/config";
-import { MissingStore } from "../utilities/errors";
+import { getConfig } from "../config/config";
+import { MissingStoreError, NotImplementedError } from "../utilities/errors";
 
 export type File = Buffer | string;
-export type Content = string | Buffer;
+export type Content = string;
 
 export interface StorageInterface {
-  put(targetPath: string, content: Content): Promise<URL>;
-  get(targetPath: string): Promise<File>;
+  put: (targetPath: string, content: Content) => Promise<URL>;
+  get?: (targetPath: string) => Promise<File>;
 }
 
 /**
  * put() takes a batch file to store with the chosen hosting solution and
- * returns the URI of the file once stored.
+ * returns the URL of the file once stored.
  *
- * @param   targetPath  The location where the file is stored
+ * @param   targetPath  The path to and name of file.
  * @param   content The file object to store on the chosen hosting solution
- * @param   opts  Any optional overrides for the default configuration settings
  * @returns       The URI of the hosted file
  */
-export const put = async (targetPath: string, content: Content, opts?: Config): Promise<URL> => {
-  const { store } = await getConfig(opts);
-  if (!store) throw MissingStore;
+export const put = async (targetPath: string, content: Content): Promise<URL> => {
+  const { store } = getConfig();
```

Comment:
let's chat tomorrow with folks about how we are using the config?

Reply:
I confess I'm not clear about this either.  This is pretty much what I thought we were supposed to be doing.